include/grpc++/security/server_credentials.h

```diff
@@ -39,6 +39,7 @@
 
 #include <grpc++/security/auth_metadata_processor.h>
 #include <grpc++/support/config.h>
+#include <grpc/grpc_security.h>
```

Comment:
I'm not super thrilled about this new top level dependency (even though I understand why it's been introduced). ctiller@ may have a good idea on how to make is such that the same enum can be consumed from both C and C++ (as the one that we use for grpc_status_code for example).